linmod/data.py

```diff
@@ -135,6 +135,71 @@
 `data` key.
 """
 
+hhs_regions = {
+    "Connecticut": "HHS-1",
+    "Maine": "HHS-1",
+    "Massachusetts": "HHS-1",
+    "New Hampshire": "HHS-1",
+    "Rhode Island": "HHS-1",
+    "Vermont": "HHS-1",
+    "New Jersey": "HHS-2",
+    "New York": "HHS-2",
+    "Puerto Rico": "HHS-2",
+    "Virgin Islands": "HHS-2",
+    "Delaware": "HHS-3",
+    "Washington DC": "HHS-3",
+    "Maryland": "HHS-3",
+    "Pennsylvania": "HHS-3",
+    "Virginia": "HHS-3",
+    "West Virginia": "HHS-3",
+    "Alabama": "HHS-4",
+    "Florida": "HHS-4",
+    "Georgia": "HHS-4",
+    "Kentucky": "HHS-4",
+    "Mississippi": "HHS-4",
+    "North Carolina": "HHS-4",
+    "South Carolina": "HHS-4",
+    "Tennessee": "HHS-4",
+    "Illinois": "HHS-5",
+    "Indiana": "HHS-5",
+    "Michigan": "HHS-5",
+    "Minnesota": "HHS-5",
+    "Ohio": "HHS-5",
+    "Wisconsin": "HHS-5",
+    "Arkansas": "HHS-6",
+    "Louisiana": "HHS-6",
+    "New Mexico": "HHS-6",
+    "Oklahoma": "HHS-6",
+    "Texas": "HHS-6",
+    "Iowa": "HHS-7",
+    "Kansas": "HHS-7",
+    "Missouri": "HHS-7",
+    "Nebraska": "HHS-7",
+    "Colorado": "HHS-8",
+    "Montana": "HHS-8",
+    "North Dakota": "HHS-8",
+    "South Dakota": "HHS-8",
+    "Utah": "HHS-8",
+    "Wyoming": "HHS-8",
+    "Arizona": "HHS-9",
+    "California": "HHS-9",
+    "Hawaii": "HHS-9",
+    "Nevada": "HHS-9",
+    "American Samoa": "HHS-9",
+    "Commonwealth of the Northern Mariana Islands": "HHS-9",
+    "Federated States of Micronesia": "HHS-9",
+    "Guam": "HHS-9",
+    "Marshall Islands": "HHS-9",
+    "Republic of Palau": "HHS-9",
+    "Alaska": "HHS-10",
+    "Idaho": "HHS-10",
+    "Oregon": "HHS-10",
+    "Washington": "HHS-10",
+}
+"""
+Dictionary form of https://www.hhs.gov/about/agencies/iea/regional-offices/index.html, except that DC is Washington DC not District of Columbia
```

Comment:
cf https://github.com/cdcent/cfa-predict-handbook/issues/304

Reply:
We inherited Washington DC from Nextstrain, I've split this off into #64 since that touches the data processing pipeline.